Gemfile

```diff
@@ -60,6 +60,7 @@ gem "secure_headers",                 "~>3.0.0"
 gem "simple-rss",                     "~>1.3.1",       :require => false
 gem "snmp",                           "~>1.2.0",       :require => false
 gem "sshkey",                         "~>1.8.0",       :require => false
+gem "thin",                           "~>1.7.0",       :require => false
```

Comment:
@simaishi Note, this is a new version of thin ^.  We need this for rack 2 support which is used by rails 5.

Thin is compiled and so is eventmachine, one of it's dependencies. Here is the dependency tree from my Gemfile.lock:

```
    thin (1.7.0)
      daemons (~> 1.0, >= 1.0.9)
      eventmachine (~> 1.0, >= 1.0.4)
      rack (>= 1, < 3)
```